### PR TITLE
feat(router/jlu): add JLU Phy

### PR DESCRIPTION
**Pull request by tsurumi-yizhou**
## Involved Issue / 该 PR 相关 Issue

None

## Example for the Proposed Route(s) / 路由地址示例

```routes
/jlu/phy/xzgz/tzgg
/jlu/phy/rcpy/bksjy/BKZX
```

## New RSS Route Checklist / 新 RSS 路由检查表
  
- [x] New Route / 新的路由
  - [x] Follows [Script Standard](https://docs.rsshub.app/joinus/advanced/script-standard) / 跟随 [路由规范](https://docs.rsshub.app/zh/joinus/advanced/script-standard)
- [ ] Anti-bot or rate limit / 反爬/频率限制
  - [ ] If yes, do your code reflect this sign? / 如果有, 是否有对应的措施?
- [x] [Date and time](https://docs.rsshub.app/joinus/advanced/pub-date) / [日期和时间](https://docs.rsshub.app/zh/joinus/advanced/pub-date)
  - [x] Parsed / 可以解析
  - [ ] Correct time zone / 时区正确
- [ ] New package added / 添加了新的包
- [ ] `Puppeteer`

### CODE REVIEW

File: lib/routes/jlu/phy/rcpy/index.ts [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    path: '/phy/rcpy/:category/:column',
+    categories: ['university'],
+    example: '/jlu/phy/rcpy/bksjy/BKZX',
+    radar: [
+        {
+            source: ['phy.jlu.edu.cn/rcpy/bksjy/BKZX'],
+        },
+    ],
+    name: '吉林大学物理学院 - 本科生培养',
+    maintainers: ['tongyz'],
```
Comment (TonyRL) [inline lib/routes/jlu/phy/rcpy/index.ts:14]:
```suggestion
    maintainers: ['tsurumi-yizhou'],
```

File: lib/routes/jlu/phy/xzgz/index.ts [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    path: '/phy/xzgz/:category',
+    categories: ['university'],
+    example: '/jlu/phy/xzgz/tzgg',
+    radar: [
+        {
+            source: ['phy.jlu.edu.cn/xzgz/tzgg'],
+        },
+    ],
+    name: '吉林大学物理学院 - 通知公告',
+    maintainers: ['tongyz'],
```
Comment (TonyRL) [inline lib/routes/jlu/phy/xzgz/index.ts:14]:
```suggestion
    maintainers: ['tsurumi-yizhou'],
```

File: lib/routes/jlu/phy/rcpy/index.ts [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+export const route: Route = {
+    path: '/phy/rcpy/:category/:column',
+    categories: ['university'],
+    example: '/jlu/phy/rcpy/bksjy/BKZX',
+    radar: [
+        {
+            source: ['phy.jlu.edu.cn/rcpy/bksjy/BKZX'],
+        },
+    ],
+    name: '吉林大学物理学院 - 本科生培养',
```
Comment (TonyRL) [inline lib/routes/jlu/phy/rcpy/index.ts:13]: No need to repeat namespace name in route name
```suggestion
    name: '物理学院 - 本科生培养',
```

File: lib/routes/jlu/phy/xzgz/index.ts [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+export const route: Route = {
+    path: '/phy/xzgz/:category',
+    categories: ['university'],
+    example: '/jlu/phy/xzgz/tzgg',
+    radar: [
+        {
+            source: ['phy.jlu.edu.cn/xzgz/tzgg'],
+        },
+    ],
+    name: '吉林大学物理学院 - 通知公告',
```
Comment (TonyRL) [inline lib/routes/jlu/phy/xzgz/index.ts:13]:
```suggestion
    name: '物理学院 - 通知公告',
```

File: lib/routes/jlu/phy/xzgz/index.ts [hunk 5]
```diff
@@ -0,0 +1,20 @@
+import { Route } from '@/types';
+import { handler } from '../handler';
+
+export const route: Route = {
+    path: '/phy/xzgz/:category',
```
Comment (TonyRL) [inline lib/routes/jlu/phy/xzgz/index.ts:5]: Missing parameter description

File: lib/routes/jlu/phy/rcpy/index.ts [hunk 6]
```diff
@@ -0,0 +1,20 @@
+import { Route } from '@/types';
+import { handler } from '../handler';
+
+export const route: Route = {
+    path: '/phy/rcpy/:category/:column',
```
Comment (TonyRL) [inline lib/routes/jlu/phy/rcpy/index.ts:5]: Missing parameter description

**Comment by tsurumi-yizhou** [comment]
added descriptions and modifiered maintainers

File: lib/routes/jlu/phy/rcpy/index.ts [hunk 7]
```diff
@@ -0,0 +1,31 @@
...
```
Comment (TonyRL) [inline lib/routes/jlu/phy/rcpy/index.ts:5]: Parameter description should be [an object](https://docs.rsshub.app/joinus/new-rss/start-code#creating-route)

File: lib/routes/jlu/phy/xzgz/index.ts [hunk 8]
```diff
@@ -0,0 +1,28 @@
...
```
Comment (TonyRL) [inline lib/routes/jlu/phy/xzgz/index.ts:5]: Parameter description should be [an object](https://docs.rsshub.app/joinus/new-rss/start-code#creating-route)

**Pull request merged**